app/components/graph-editor/drawable-interfaces.ts

```diff
@@ -61,6 +55,24 @@ export interface DrawableGraph {
     readonly edges: Iterable<DrawableEdge>;
 
     /**
+     * selectedItems  
+     *   The set of selected graph elements.
+     */
+    readonly selectedItems: Set<Drawable>;
```

Comment:
I'm not sure if it's really enforced, but I put it there so tsc can hopefully catch me if I try to set it's value as opposed to just modifying the collection.